### PR TITLE
Feature/layout update

### PR DESCRIPTION
**Pull request by dancer2090**
This PR includes a new layout https://projects.invisionapp.com/d/main#/console/18526489/412279596/inspect

Changes:
1) Disabled search block and subscribe block
2) Updated the route control
3) For mobiles added a modal window with  no-responsive text

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -57,3 +57,91 @@ export const PERCIPITATION_RAMP_COLORS = ['#A6BDDB', '#3690C0', '#034E7B']; // [
...
+    link: '/press-releases',
+    fullUrl: 'https://climate.copernicus.eu/press-releases'
+  },
+  {
+    title: 'Trends',
+    link: '/complete-list-tenders-issued-c3s',
+    fullUrl: 'https://climate.copernicus.eu/complete-list-tenders-issued-c3s'
+  },
+  {
+    title: 'Help &amp; Support',
```
Comment (dfrico) [inline src/constants.js]: Please check the rendering of this HTML entity, it's rendering as "Help & amp; Support" in the browser.

File: src/constants.js [hunk 2]
```diff
@@ -57,3 +57,91 @@ export const PERCIPITATION_RAMP_COLORS = ['#A6BDDB', '#3690C0', '#034E7B']; // [
...
+  },
+  {
+    title: 'Data licence',
+    link:
+      'https://climate.copernicus.eu/sites/default/files/2018-04/20170117_Copernicus_License_V1.0.pdf',
+    fullUrl:
+      'https://climate.copernicus.eu/sites/default/files/2018-04/20170117_Copernicus_License_V1.0.pdf'
+  },
+  {
+    title: 'Disclaimer &amp; Privacy',
```
Comment (dfrico) [inline src/constants.js]: Same issue here

File: src/styles/_base.scss [hunk 3]
```diff
@@ -6,18 +6,352 @@ body {
...
+  }
+}
+
+@media screen and (min-width: 360px) {
+  h3--alt, .h3--alt {
+    line-height: calc(23px + 5 * (100vw - 360px) / 840);
+  }
+}
+
+@media screen and (min-width: 1200px) {
```
Comment (dfrico) [inline src/styles/_base.scss:348]: Please review the 1200px media query, the header text is cut (i.e. "BIOCLIMATIC VARIAB"). I know we're not supporting mobile, but I don't think 1200px should be an issue for that.

**Review by dfrico — CHANGES_REQUESTED** [review]
Hello! I left you some comments to address regarding the code, I'll also send you some screenshots for the UI issues I could find.

File: src/constants.js (hunk 2 above)
Comment (dancer2090) [reply src/constants.js]: Fixed here: https://github.com/Vizzuality/copernicus-forest/pull/33/commits/d76cf8d043b4f4a776994fc7d880fe0791eefec5

File: src/constants.js (hunk 1 above)
Comment (dancer2090) [reply src/constants.js]: Fixed here: https://github.com/Vizzuality/copernicus-forest/pull/33/commits/d76cf8d043b4f4a776994fc7d880fe0791eefec5

File: src/styles/_base.scss (hunk 3 above)
Comment (dancer2090) [reply src/styles/_base.scss:348]: Fixed here: https://github.com/Vizzuality/copernicus-forest/pull/33/commits/d76cf8d043b4f4a776994fc7d880fe0791eefec5

**Review by dfrico — APPROVED** [review]
Looks good to me 👍 nice job

**Pull request merged**